app/bundles/LeadBundle/Entity/LeadRepository.php

```diff
@@ -562,16 +562,23 @@ public function getEntitiesDbalQueryBuilder()
 
     /**
      * @param $order
+     * @param mixed[] $args
      *
      * @return \Doctrine\ORM\QueryBuilder
      */
-    public function getEntitiesOrmQueryBuilder($order)
+    public function getEntitiesOrmQueryBuilder($order, array $args=[])
     {
-        $alias = $this->getTableAlias();
-        $q     = $this->getEntityManager()->createQueryBuilder();
-        $q->select($alias.', u, i,'.$order)
+        $alias  = $this->getTableAlias();
+        $select = [$alias, 'u', $order];
+        $q      = $this->getEntityManager()->createQueryBuilder();
+
+        if (!empty($args['joinIpAddresses'])) {
```

Comment:
I think that in order to keep BC then the `if` should be true if the `$args['joinIpAddresses']` does not exist. So something like
```suggestion
        if (!isset($args['joinIpAddresses']) || true === $args['joinIpAddresses']) {
```

Reply:
@escopecz I changed this code a bit, because we can have a join without a select before, so tests were failing